cypress/component/date/DateEditor.spec.tsx

```diff
@@ -15,93 +21,63 @@ const MONTHS = {
   11: 'Dec',
 };
 
-describe('Date Editor', () => {
-  const now = new Date();
-
-  const selectors = {
-    getDateInput: () => {
-      return getIframe().findByTestId('cf-ui-datepicker-input');
-    },
-    getCalendarTrigger: () => {
-      return getIframe().findByTestId('cf-ui-datepicker-button');
-    },
-    getTimeInput: () => {
-      return getIframe().findByTestId('time-input');
-    },
-    getTimezoneInput: () => {
-      return getIframe().findByTestId('timezone-input');
-    },
-    getClearBtn: () => {
-      return getIframe().findByTestId('date-clear');
-    },
-    getCalendar: () => {
-      return getIframe().find('.rdp');
-    },
-    getCalendarMonth: () => {
-      return getIframe().find('.rdp select[aria-label="Month: "]');
-    },
-    getCalendarYear: () => {
-      return getIframe().find('.rdp select[aria-label="Year: "]');
-    },
-    getCalendarTodayDate: () => {
-      return getIframe().find('.rdp .rdp-day_today');
-    },
-    getCalendarSelectedDate: () => {
-      return getIframe().find('.rdp .rdp-day_selected');
-    },
-  };
-
-  const getToday = () => {
-    const month = now.getMonth();
-    const year = now.getFullYear();
-    const date = now.getDate();
-    return {
-      month,
-      year,
-      date,
-    };
-  };
-
-  const getTodayShortString = () => {
-    const date = `${getToday().date}`.padStart(2, '0');
-    const month = MONTHS[getToday().month];
-    const year = getToday().year;
+const now = new Date();
+
+const selectors = {
+  getDateInput: () => cy.findByTestId('cf-ui-datepicker-input'),
+  getCalendarTrigger: () => cy.findByTestId('cf-ui-datepicker-button'),
+  getTimeInput: () => cy.findByTestId('time-input'),
+  getTimezoneInput: () => cy.findByTestId('timezone-input'),
+  getClearBtn: () => cy.findByTestId('date-clear'),
+  getCalendar: () => cy.get('.rdp'),
+  getCalendarMonth: () => cy.get('.rdp select[aria-label="Month: "]'),
```

Comment:
Wondering if we could use `cy.findByLabel('Month: ')` instead 🤔 
Same for year
Then we would also test the a11y :)